ml-agents/mlagents/trainers/tf.py

```diff
@@ -0,0 +1,19 @@
+import tensorflow as tf
+
+# TODO better version check, this will do for now though
+is_tf2 = tf.__version__ == "2.0.0"
```

Comment:
There's definitely better ways to do this, just wanted something up and running initially.